PayItForward.Data.Services/Abstraction/IStoriesService.cs

```diff
@@ -1,12 +1,17 @@
-﻿namespace PayItForward.Services.Data.Abstraction
+﻿namespace PayItForward.Services.Abstraction
 {
+    using System;
     using System.Collections.Generic;
     using PayItForward.Models;
 
     public interface IStoriesService
     {
-        IEnumerable<StoryDTO> GetStories(int take, int skip, string containsTitle = "");
+        IEnumerable<StoryDTO> GetStories(int take, int skip, string containsTitle = "", string isFromCategory = "");
 
-        int CountStories(string containsTitle = "");
+        IEnumerable<StoryDTO> GetStories();
+
+        int CountStories(string containsTitle = "", string isFromCategory = "");
```

Comment:
Contains title can be changed to titleSubstring and isFromCategory to CategoryName or Category. Please review all the places where you have this, so to not put the same comment on all the places